### ISSUE TITLE
Integrate into official agent

### ISSUE THREAD

**Issue by mikker** [kressh/elastic-apm-grape-middleware#1]
Hi! Just stumbled upon this – it's great!

Do you mind if I integrate it or a version of it in the official agent? You're very welcome to submit a PR if you want the contribution credit 😃

**Comment by bsedin** [comment]
Hi, @mikker!
You are too generous, I just modified your original middleware little bit :) But thanks!
I'd love to contribute, give me couple hours.

**Comment by mikker** [comment]
Nice! I also saw your custom Rails middleware. You use that mainly to have endpoints show up as `GET /somewhere`? Correct?

**Comment by bsedin** [comment]
Yep. There are some weird calculations of route pattern to group transactions like `GET /posts/:post_id/comments/:id`

**Comment by bsedin** [comment]
https://github.com/elastic/apm-agent-ruby/pull/119
